command/user.go

```diff
@@ -120,37 +107,31 @@ var PasswordUser = cli.Command{
 		},
 	},
 	Action: func(c *cli.Context) error {
-		m, cfg := setup(c)
+		client := esetup(c, AuthUsersValidation)
+		session := client.Session().Get()
 
 		username := c.String("user")
 		password := c.String("password")
 		currentPassword := c.String("current-password")
 
-		session, err := m.GetSession(cfg.Token)
-		if err != nil {
-			h.PrintError("You don’t have permissions to perform this action")
-		}
-
 		if !session.IsAdmin() && username != "" {
 			h.PrintError("You don’t have permissions to perform this action")
 		}
 
 		if session.IsAdmin() && username != "" {
+
 			if password == "" {
 				h.PrintError("Please provide a valid password for the user with `--password`")
 			}
-
-			// Just change the password with the given values for the given user
-			if err = m.ChangePasswordByAdmin(cfg.Token, username, password); err != nil {
-				h.PrintError(err.Error())
-			}
+			user := client.User().Get(username)
```

Comment:
As you can see here it's pretty easy to update a user, `manager.Client` will be taking care of printing errors in case something is broken, so its pretty easy to follow whats going on by only reading the code on the commands